### PR TITLE
[PM-12505] - add delete send button to footer

### PR DESCRIPTION
**Pull request by jaasen-livefront**
## 🎟️ Tracking
https://bitwarden.atlassian.net/browse/PM-12505

## 📔 Objective

This PR adds the delete button to the footer of the edit send page.

## 📸 Screenshots

https://github.com/user-attachments/assets/6d1e657e-7816-4564-b5d5-58311b2c92dc

## ⏰ Reminders before review

- Contributor guidelines followed
- All formatters and local linters executed and passed
- Written new unit and / or integration tests where applicable
- Protected functional changes with optionality (feature flags)
- Used internationalization (i18n) for all UI strings
- CI builds passed
- Communicated to DevOps any deployment requirements
- Updated any necessary documentation (Confluence, contributing docs) or informed the documentation team

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

File: apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.ts [hunk 1]
```diff
@@ -92,6 +104,37 @@ export class SendAddEditComponent {
     this.location.back();
   }
 
+  async deleteSend() {
+    const confirmed = await this.dialogService.openSimpleDialog({
+      title: { key: "deleteSend" },
+      content: { key: "deleteSendConfirmation" },
```
Comment (djsmith85) [inline apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.ts:110]: The message needs to be changed according to the design into:
"Are you sure you want to _permanently_ delete this Send?"

While this is being changed, it should also be used within the `send-list-items-container.component`

File: apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html [hunk 2]
```diff
@@ -13,5 +13,14 @@
     <button bitButton type="submit" form="sendForm" buttonType="primary" #submitBtn>
       {{ "save" | i18n }}
     </button>
+    <button
+      *ngIf="config?.mode !== 'add'"
+      type="button"
+      buttonType="danger"
+      class="tw-border-0 tw-ml-auto bwi-lg"
```
Comment (djsmith85) [inline apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html:20]: The classes shouldn't be need

File: apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html [hunk 3]
```diff
@@ -13,5 +13,14 @@
...
+      bitIconButton="bwi-trash"
+      (click)="deleteSend()"
```
Comment (djsmith85) [inline apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html:22]: Please use `bitAction` instead. Example for the delete button can be found on the [vault's view2 component](https://github.com/bitwarden/clients/blob/bcddf575a1935e2c29189f0707587afaf82f5f84/apps/browser/src/vault/popup/components/vault-v2/view-v2/view-v2.component.html#L29)

**Review by djsmith85 — CHANGES_REQUESTED** [review]
As a quick win (approaching deadline), we can include the delete within the send-add-edit.component. 

Thinking forward though about re-using the `send-form.component` on other clients, it should take care of the deletion (via `SendFormConfigService`) and prompt like it already does for the submit action.

I envision a `onSendDeleted`-event which would then take care of the navigation and success toast, which are client-specific. On browser I'd navigate back (this.location.back) but on web I'd close a dialog and on desktop I might remove/hide the tools-send-form 🤷

File: apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html (hunk 2 above)
Comment (jaasen-livefront) [reply apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html:20]: @djsmith85 Without the classes the icon gets a border, doesn't align right, and is too small.

Comment (djsmith85) [reply apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html:20]: @danielleflinn

**Review by djsmith85 — APPROVED** [review]
Approving for the changes I requested, leaving it to @danielleflinn to comment/approve on the usage of the classes.

**Review by djsmith85 — CHANGES_REQUESTED** [review]

File: apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html (hunk 2 above)
Comment (danielleflinn) [reply apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html:20]: This component should use https://components.bitwarden.com/?path=/story/component-library-icon-button--danger, in the current styles it has a border which will be removed in the Design System feature branch.

Comment (jaasen-livefront) [reply apps/browser/src/tools/popup/send-v2/add-edit/send-add-edit.component.html:20]: Ok I'll remove the `tw-border-0` class. According to @danielleflinn's feedback everything else should remain unless I'm missing something.

**Review by danielleflinn — APPROVED** [review]

**Review by djsmith85 — APPROVED** [review]

**Pull request merged**